manifests/config.pp

```diff
@@ -232,6 +224,7 @@
     group   => $prometheus::server::group,
     purge   => true,
     recurse => true,
+    notify  => Class['prometheus::service_reload'], # After purging, a reload is needed
```

Comment:
Previously, a purge of scrape job files would have caused a full service restart.